GPU/GPUTracking/Interface/GPUO2Interface.cxx

```diff
@@ -103,10 +104,11 @@ int GPUTPCO2Interface::RunTracking(GPUTrackingInOutPointers* data)
 
 void GPUTPCO2Interface::Clear(bool clearOutputs) { mRec->ClearAllocatedMemory(clearOutputs); }
 
-void GPUTPCO2Interface::GetClusterErrors2(int row, float z, float sinPhi, float DzDs, float& ErrY2, float& ErrZ2) const
+void GPUTPCO2Interface::GetClusterErrors2(int row, float z, float sinPhi, float DzDs, short clusterState, float& ErrY2, float& ErrZ2) const
 {
   if (!mInitialized) {
```

Comment:
Do we really need this silent check at your very call?

Reply:
although independent from this PR, I am okay with removing it. Obviously it will lead to a segfault when used without initializing